servers/rendering/renderer_rd/shaders/volumetric_fog.glsl

```diff
@@ -166,17 +166,17 @@ void main() {
 	vec4 local_pos = params.transform * world;
 	local_pos.xyz /= local_pos.w;
 
-	float sdf = -1.0;
+	float extents = params.size / 2.0 float sdf = -1.0;
```

Comment:
```suggestion
	float extents = params.size / 2.0;
	float sdf = -1.0;
```

Reply:
Ohh, of course - I wondered why the CI wanted the lines put after each other, but obviously I forgot the semicolon (;
I'll do some tests now.